src/EventGrid/06-15-2022_design_review.md

```diff
@@ -0,0 +1,1067 @@
+
+If you just add parameter to existing cmdlets and parameter definition complies with guideline, the design review is not expected.
+
+- Have you read above statement?
+
+    - `Yes`
+
+## Service Release Details
+
+- Is this an Embargoed Preview, A Public Preview, or a General Release?
+
+    - `General Release`
+
+- What is the expected service release date?
+
+    - `06-15-2022`
+    
+- Which [Powershell module](https://github.com/Azure/azure-powershell/blob/main/documentation/azure-powershell-modules.md) are these changes being made in?
+
+    - `Az.EventGrid`
+
+## Contact Information
+
+- Main developer contacts (emails + github aliases)
+
+    - `Brandon Neff: brandonneff@microsoft.com, Brandon411`
+
+- PM contact (email + github alias) 
+
+    - `Roberto Cervantes Rivero: robece@microsoft.com`
+    - `Javier Fernandez: jafernan@microsoft.com`
+
+- Other people who should attend a design review (email)
+
+    - `Ashraf Hamad: ahamad@microsoft.com`
+    - `Kishore Kumar Peskhar: kishp@microsoft.com`
+
+## High Level Scenarios
+
+- Describe how your feature is intended to be used by customers.
+
+    - [Partner Events overview for customers - Azure Event Grid | Microsoft Docs](https://docs.microsoft.com/en-us/azure/event-grid/partner-events-overview)
+
+    - Azure Event Grid's **Partner Events** allows customers to **subscribe to events** that originate in a registered system using the same mechanism they would use for any other event source on Azure, such as an Azure service. Those registered systems integrate with Event Grid are known as "partners".
+
+      This feature also enables customers to **send events** to partner systems that support receiving and routing events to customer's solutions/endpoints in their platform. Typically, partners are software-as-a-service (SaaS) or [ERP](https://en.wikipedia.org/wiki/Enterprise_resource_planning) providers, but they might be corporate platforms wishing to make their events available to internal teams.
+
+      They purposely integrate with Event Grid to realize end-to-end customer use cases that end on Azure (customers subscribe to events sent by partner) or end on a partner system (customers subscribe to Microsoft events sent by Azure Event Grid). Customers bank on Azure Event Grid to send events published by a partner to supported destinations such as webhooks, Azure Functions, Azure Event Hubs, or Azure Service Bus, to name a few.
+
+- Piping scenarios / how these cmdlets are used with existing cmdlets
+
+    - `{ SCENARIOS OUTLINED HERE }`
+
+- Sample of end-to-end usage
+
+    - Please provide comprehensive examples that don't assume additional setup. It helps the audience understand your feature.
```

Comment:
plz make sure to add example here.

example,
user create partner configuration for specific SubId/RG and add initial publisher to approved/trusted publishers
user authorize or unauthorize specific publisher by immutableid

user create namespace then channel which in return create partner topic
user activate partner topic and create event subscriptioon